docs/docs/reactions/conrete-stores.mdx

```diff
@@ -0,0 +1,136 @@
+---
+sidebar_position: 1
+---
+
+# Concrete stores
+
+Equinox stores expose a `source` package to wire reactions up. In the [previous
+section](/docs/reactions) we created a `handler`. Source packages interact with
+`Sink`s rather than handler functions so we must first construct a sink from
+the handler.
+
+```ts
+import { StreamsSink } from "@equinox-js/propeller"
+const sink = StreamsSink.create({
+  handler,
+  // How many streams are we OK to process concurrently?
+  maxConcurrentStreams: 10,
+  // How many batches can be pre-loaded beyond what's been completed 
+  // (checkpointing is always sequential in order of age)
+  maxReadAhead: 3,
+})
+```
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs>
```

Comment:
This comes out spiffy
<img width="884" alt="Screenshot 2023-09-23 at 23 00 45" src="https://github.com/equinox-project/equinox-js/assets/3721521/da7aa270-3be9-4cc5-b993-bbcf1ba277d4">